scripts/hid_configurator/README.rst

```diff
@@ -49,6 +49,12 @@ Complete the following steps:
 
 1. Download the HIDAPI library from `HIDAPI releases`_.
    Use the bundled DLL or build it according to instructions in `HIDAPI library`_.
+   To install the DLL, select the appropriate HIDADPI library version for the used Windows system (either ``x86`` or ``x64``).
+   Next, paste :file:`hidapi.dll` and `hidapi.lib` files into either:
```

Comment:
```suggestion
Next copy the :file:`hidapi.dll` file and paste it into either:
```

Reply:
I think we do not need lib file at all here

Reply:
Yea, DLL only should be sufficient